web/src/components/MalteHelpForm.tsx

```diff
@@ -0,0 +1,242 @@
+  helpButtonOffer: OfferModel
+}
+
+const MalteHelpForm = ({ languageCode, cityCode, helpButtonOffer }: MalteHelpFormProps): ReactElement => {
+  const { t } = useTranslation('malteHelpForm')
+  const [sendingStatus, setSendingStatus] = useState<SendingStatusType>('idle')
+  const [submitted, setSubmitted] = useState(false)
+  const [contactChannel, setContactChannel] = useState<ContactChannel>('email')
+  const [email, setEmail] = useState('')
+  const [telephone, setTelephone] = useState('')
+  const [name, setName] = useState('')
+  const [roomNumber, setRoomNumber] = useState('')
+  const [contactType, setContactType] = useState<ContactGender>('any')
+  const [comment, setComment] = useState('')
+  const missingData =
+    !name.length ||
+    (!email.length && contactChannel === 'email') ||
+    (!telephone.length && contactChannel === 'telephone')
+  const dashboardRoute = cityContentPath({ languageCode, cityCode })
+
+  const submitHandler = useCallback(
+    (event: SyntheticEvent<HTMLFormElement>) => {
+      const form = event.currentTarget
+      if (!form.checkValidity()) {
+        // event.preventDefault()
+        event.stopPropagation()
+        const invalidInput = form.querySelector(':invalid:not(fieldset)')
+        invalidInput?.scrollIntoView({ behavior: 'smooth' })
+        setSendingStatus('idle')
+      } else {
+        setSendingStatus('sending')
+        const request = async () => {
+          await submitHelpForm({ cityCode, languageCode, helpButtonOffer })
+          setSendingStatus('successful')
+        }
+
+        request().catch(err => {
+          reportError(err)
+          event.preventDefault()
+          setSendingStatus('failed')
+        })
+      }
+    },
+    [cityCode, helpButtonOffer, languageCode],
+  )
+
+  if (sendingStatus === 'successful') {
+    return (
+      <Container>
+        <div>{t('submitSuccessful')}</div>
+        <Link to={dashboardRoute}>{t('error:goTo.categories')}</Link>
+      </Container>
+    )
+  }
+
+  return (
+    <>
+      <Note>
+        <StyledIcon src={SupportIcon} direction={config.getScriptDirection(languageCode)} />
+        {t('supportNote')}
+      </Note>
+      <Note>
+        <StyledIcon src={SecurityIcon} direction={config.getScriptDirection(languageCode)} />
+        {t('securityNote')}
+      </Note>
+      <Form onSubmit={submitHandler}>
+        <Input
+          id='name'
+          hint={t('name')}
+          hintIsLabel
+          required
+          value={name}
+          direction={config.getScriptDirection(languageCode)}
+          onChange={setName}
+          submitted={submitted}
+        />
+        <Input
+          id='roomNumber'
+          hint={`${t('roomNumber')} (${t('common:optional')})`}
+          hintIsLabel
+          value={roomNumber}
+          direction={config.getScriptDirection(languageCode)}
+          onChange={setRoomNumber}
+          submitted={submitted}
+        />
+        <RadioGroup caption={t('howToBeContacted')}>
```

Comment:
RadioGroup should have `role="radiogroup"`
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role